source/features/remove-actions-tab.tsx

```diff
@@ -0,0 +1,41 @@
+import cache from 'webext-storage-cache';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getRepo} from '../github-helpers';
+import * as api from '../github-helpers/api';
+
+const getWorkflowsCount = cache.function(async (): Promise<number> => {
+	const {workflows} = await api.v3('actions/workflows');
+	return workflows.length;
+}, {
+	maxAge: {days: 1},
+	cacheKey: (): string => __filebasename + ':' + getRepo()!.nameWithOwner
+});
+
+async function removeActionsTab(): Promise<void | false> {
+	const actionsTab = await elementReady('[data-hotkey="g a"]');
+
+	if (
+		!actionsTab || // Actions Tab does not exist 🎉
+		actionsTab.matches('.selected') || // User is on Actions tab 👀
+		await getWorkflowsCount() > 0 // Actions Workflows are configured for the repository
+	) {
+		return false;
+	}
+
+	actionsTab.remove();
+}
+
+void features.add(__filebasename, {
+	include: [
+		pageDetect.isRepo
+	],
+	exclude: [
+		// Repo owners should see the tab. If they don't need it, they should disable actions altogether
+		pageDetect.canUserEditRepo
```

Comment:
You can't disable actions. Please ensure that every line still applies when copy-pasting code.

The exclusion is fine, but the comment is not applicable

Reply:
https://github.com/yakov116/TestR

![image](https://user-images.githubusercontent.com/16872793/119718793-fcdfc680-be35-11eb-9d66-4abbefcdb282.png)

I suggested that code.

Reply:
TIL